### PR TITLE
bors.yaml: re-activate labels check + add block_labels

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
Reactivates the `check_labels` action (on `review`) and adds to block on "Process: missing approvals".

### Testing procedure
Let's add the >1 ACK label for testing once #19160 was merged.

### Issues/PRs references
Requires https://github.com/RIOT-OS/RIOT/pull/19160

### CODE REVIEW

**Review by benpicco — APPROVED** [review]

**Comment by miri64** [comment]
(just for testing)

**Comment by miri64** [comment]
Mhh... bors has put the PR in its queue, even though the status indicates failure and the approval was dismissed oO. Ok, let's see what it does with that input ...

File: bors.toml [hunk 1]
```diff
@@ -19,6 +18,8 @@ status = [
   "tools-build-success",
   ]
 
+block_labels = "Process: missing approvals"
```
Comment (kaspar030) [inline bors.toml:21]: this maybe needs to be a list.

```suggestion
block_labels = [ "Process: missing approvals" ]
```

**Comment by kaspar030** [comment]
bors cancel
bors merge

**Comment by kaspar030** [comment]
that looks more like it.

**Comment by miri64** [comment]
Huh, why is this not merged yet?

**Comment by miri64** [comment]
bors merge

**Review by maribu — APPROVED** [review]

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
bors merge

**Review by miri64 — COMMENTED** [review]
Grml..

**Comment by miri64** [comment]
bors retry

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
(in the current state this can lead to race condition where the check-labels action did not run through yet though it would set the `missing approvals` label, but let's see what it does with that input)

**Comment by miri64** [comment]
bors retry

**Comment by miri64** [comment]
bors cancel

**Comment by miri64** [comment]
Returning to previous state...

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
(ノ^_^)ノ┻━┻

**Comment by miri64** [comment]
bors retry

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
Let's not halt the merge train for this small change

bors cancel
bors merge

**Comment by miri64** [comment]
bors merge

**Comment by miri64** [comment]
Come on bors do all four waiting PRs!

bors cancel
bors merge

**Pull request merged**